src/pytkdocs/parsers/docstrings/numpy.py

```diff
@@ -89,11 +89,16 @@ def read_parameters_section(
                 if signature_param.default is not empty:
                     default = signature_param.default
                 kind = signature_param.kind
+            
+            description = param.description if param.description else ""
```

Comment:
```suggestion
            description = param.description or ""
```

Reply:
Ah, I thought there should be a easier way to say that. Thanks for the the tip.